From d91baefd5310711de1f4ff7f4f7a367870c3ea4e Mon Sep 17 00:00:00 2001
From: Martin Abente Lahaye <tch@sugarlabs.org>
Date: Fri, 3 Jan 2020 11:46:23 -0300
Subject: [PATCH 2/5] Fix card-image for sound cards

There was a typo in the svg file name, it's actually
called "sounds.svg".
---
 createcardpanel.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/createcardpanel.py b/createcardpanel.py
index 86879f8..688ff14 100644
--- a/createcardpanel.py
+++ b/createcardpanel.py
@@ -452,9 +452,9 @@ class CardEditor(Gtk.EventBox):
 
             # add a icon too
             sound_icon_path = join(activity.get_bundle_path(),
-                                   'icons/sound.svg')
+                                   'icons/sounds.svg')
             destination_path = join(self._game.model.data['pathimg'],
-                                    'sound.svg')
+                                    'sounds.svg')
             shutil.copy(sound_icon_path, destination_path)
 
             self.card.set_image_path(destination_path)
-- 
2.23.0

